src/improvements/tasks/sep/004-opcm-upgrade-v300/README.md

```diff
@@ -0,0 +1,41 @@
+# 004-opcm-upgrade-v300: Sepolia OPCM v3.0.0: op, soneium, ink
+
+Status: [DRAFT]()
+
+## Objective
+
+<!-- TODO(#740) - Add a link to the gov post -->
+Executes [Upgrade 14]() on Sepolia for the OP, Soneium, and Ink chains.
+
+In summary, this task uses `op-contract/v3.0.0` [OPContractsManager](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v3.0.0-rc.2/packages/contracts-bedrock/src/L1/OPContractsManager.sol) to upgrade 3 chains:
+1. OP Sepolia Testnet
+2. Soneium Testnet Minato
+3. Ink Sepolia
+
+### Timing
+
+Expected to be executed on or around 2025-04-09.
+
+## Transaction creation
+
+The transaction is generated by the [OPCMV300 template script](../../../template/OPCMUpgradeV300.sol),
+which reads the inputs from the [`config.toml`](./config.toml) file.
+
+## Signing and execution
+
+Follow the instructions in the [Nested Execution](../../../NESTED.md) guide for the following steps:
+
+- [1. Update repo](../../../NESTED.md#1-update-repo)
+- [2. Setup Ledger](../../../NESTED.md#2-setup-ledger)
+- [3. Simulate and validate the transaction](../../../NESTED.md#3-simulate-and-validate-the-transaction)
+
+Then follow the instructions in the [Validation](./VALIDATION.md) guide.
+
+## Simulation
+
+When simulating, ensure the logs say `Using script <your_path_to_superchain_ops>/superchain-ops/src/improvements/template/OPCMUpgradeV200.sol`.
+Navigate to the correct task directory then run the simulate command.
+```
+cd 004-opcm-upgrade-v300
```

Comment:
Need to make sure this execute and works properly. Run it and check the tenderly simulation is good.